zio/zngio/writer.go

```diff
@@ -9,8 +9,10 @@ import (
 	"github.com/pierrec/lz4/v4"
 )
 
-// DefaultLZ4BlockSize is a reasonable default for the WriterOpts.LZ4BlockSize.
-const DefaultLZ4BlockSize = 16 * 1024
+const (
+	DefaultLZ4BlockSize     = 16 * 1024 // a reasonable default for WriterOpts.LZ4BlockSize
+	DefaultStreamRecordsMax = 5000      // a reasonable default for WriterOpts.StreamRecordsMax
```

Comment:
Nit: Make these doc comments.
```suggestion
	// DefaultLZ4BlockSize is a reasonable default for WriterOpts.LZ4BlockSize.
	DefaultLZ4BlockSize     = 16 * 1024
	// DefaultStreamRecordsMax is a reasonable default for WriterOpts.StreamRecordsMax.
	DefaultStreamRecordsMax = 5000
```